Base/Python/slicer/util.py

```diff
@@ -194,8 +194,7 @@ def lookupTopLevelWidget(objectName, verbose = True):
     if hasattr(w,'objectName'):
       if w.objectName == objectName: return w
   if verbose:
-    import sys
-    print("Failed to obtain reference to '%s'" % objectName, file=sys.stderr)
+    raise RuntimeError("Failed to obtain reference to '%s'" % objectName)
```

Comment:
Now we raise an exception. It should probably be done all the time and not only when verbose is True.